spec/controllers/people_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+# spec/controllers/people_controller_spec.rb
+require 'rails_helper'
+
+RSpec.describe PeopleController, type: :controller do
+  describe "GET #index" do
+    subject { get :index }
+
+    it "renders the template" do
+      expect(subject).to render_template(:index)
+    end
+    it "loads all of the people into @people" do
+      expect(subject.body.length).to be_truthy
```

Comment:
This is some what a weird way to phrase this. 
I would test that the response code is a success.